src/css/components/task-file-download.less

```diff
@@ -0,0 +1,9 @@
+/* ==============
+ * Task File Download (Link)
+ * ==============
+ */
+.task-file-download {
+  &.loading  {
+    // @todo: add some styles indicating that somethig is going on
```

Comment:
Agreed. I suggest to move the loading states to a separate PR and concentrating on integrating comments on this one ;)